src/app/courses/courses.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="courses">
     <div class="container">
-        <app-search></app-search>
+        <app-search (filterCourses)="onFilter($event)"></app-search>
```

Comment:
I'd not recomment tieing a common `AppSearch` componts with courses. It'd be better to make this output `searchSubmit` or something like that. Eventually, handler might be `onSearchSubmit($event)`